game-news-app/pages/api/news.js

```diff
@@ -0,0 +1,25 @@
+import axios from "axios";
+
+export default async function handler(req, res) {
+  if (req.method === "GET") {
+    const options = {
+      method: "GET",
+      url: "https://gaming-news.p.rapidapi.com/news",
+      headers: {
+        "x-rapidapi-host": "gaming-news.p.rapidapi.com",
+        "x-rapidapi-key": process.env.NEXT_PUBLIC_RAPIDAPI_KEY,
+      },
+    };
+
+    axios
```

Comment:
Use `await` if you are using [`async`](https://github.com/RapidAPI/DevRel-Examples-External/pull/37/files#diff-c5198051cca4043fbac24718bb2a7e08cb1d9847470f4688fb70f77cb81c50ddR3).

Reply:
I have removed `async`. Otherwise, the code snippet will contradict the one that is in RapidAPI Hub.